src/ex1/script.js

```diff
@@ -0,0 +1,381 @@
+const TYPE_DEADLINE = 1
+const TYPE_NOTCOMPLETE = 2
+const TYPE_FINISHED = 3
+const TYPE_DEADLINE_FINISHED = 4
+const SORT_ALPHABATICALLY = 1;
+const SORT_DEADLINE = 2;
+const SORT_NOTCOMPLETE = 3;
+const SORT_FINISHED = 4;
+
+let TaskList = [];
```

Comment:
it's confusing, I though in the first place that TaskList is a class. rename to `tasksList` to match the javascript variable conventions